documents/openapi/openapi.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   title: VA Notify API Documentation
-  version: 1.1.3
+  version: 1.2.3
```

Comment:
If  you are upping the minor version, you should probably reset the patch version. `1.2.0`

Reply:
Well - approved assuming this changes.

Reply:
Good point.  Thank you.